source/_components/sensor.rova.markdown

```diff
@@ -25,6 +25,7 @@ sensor:
   - platform: rova
     zip_code: ZIP_CODE
     house_number: HOUSE_NUMBER
+    house_number_suffix: HOUSE_NUMBER_SUFFIX
```

Comment:
You can remove this, in the first config only required variables

Reply:
Updated, thanks!